### ISSUE TITLE
virt-manager version 2.1.0 broken on VM creation

### ISSUE THREAD

**Issue by mrkz** [clearlinux/distribution#782]
On version **29560** when trying to create a new VM I get:
```
Error launching details: 'NoneType' object has no attribute 'conn'

Traceback (most recent call last):
  File "/usr/share/virt-manager/virtManager/details.py", line 372, in get_instance
    key = "%s+%s" % (vm.conn.get_uri(), vm.get_uuid())
AttributeError: 'NoneType' object has no attribute 'conn'
```

**Comment by bryteise** [comment]
I upgraded virt-manager recently and that is likely the cause. Seems like we have a test gap btw, would you be able to get your usage added?

**Comment by mrkz** [comment]
@bryteise will look into that as well.

**Comment by andyross** [comment]
This is exactly the behavior I'm seeing.  Note that it doesn't seem related to the guest config -- I've tried it with a raw system on a pre-existing empty image, qcow2 and raw images, nat and bridged networking, with and without and attached cdrom.  Same behavior every time.

Also note that there's nothing wrong with execution of existing images in libvirtd.  My old VMs are still running fine, I just can't create new ones.

**Comment by mrkz** [comment]
@andyross Thanks for the report.

We'll get virt-manager into a working state (downgrade) while I get this fixed.

Hopefully this is not a problem while we get new version working.

**Comment by mrkz** [comment]
@andyross release **29590** (or later) should have virt-manager back to a working state.

I will update the issue title to reflect the pending changes to effectively update virt-manager to latest version and keep this issue open in the meantime

**Comment by mrkz** [comment]
Got this to be fixed now.

In summary, there was an issue in GTK running in Xorg which prevented the cursor to be changed (virt-manager crashed when changing cursor).

Will close once the fix is available via.

**Comment by andyross** [comment]
Is there an open bug for the cursor issue?  Is this the same root issue that prevents some gtk apps (e.g. gnome-terminal) from correctly displaying the resize cursor at the window boundary?

**Comment by mrkz** [comment]
> Is there an open bug for the cursor issue? 
@andyross There's no issue open mentioning explicitly this, but there's some underlying issues (this one, #572 and #642) which happen to be caused by the same problem.

>  Is this the same root issue that prevents some gtk apps (e.g. gnome-terminal) from correctly displaying the resize cursor at the window boundary?
yes, this fix also fixes #572
